jsonRepresentations/med2img.json

```diff
@@ -1,134 +1 @@
 "inputFileSubStr", "type": "str", "optional": true, "flag": "--inputFileSubStr", "short_flag": "--inputFileSubStr", "action": "store", "help": "input file substring to tag a file in the inputDir", "default": "", "ui_exposed": true}, {"name": "outputFileStem", "type": "str", "optional": true, "flag": "--outputFileStem", "short_flag": "-o", "action": "store", "help": "output file stem name (with optional extension)", "default": "sample", "ui_exposed": true}, {"name": "outputFileType", "type": "str", "optional": true, "flag": "--outputFileType", "short_flag": "-t", "action": "store", "help": "output image file format", "default": "", "ui_exposed": true}, {"name": "sliceToConvert", "type": "str", "optional": true, "flag": "--sliceToConvert", "short_flag": "-s", "action": "store", "help": "slice to convert (for 3D data)", "default": "-1", "ui_exposed": true}, {"name": "frameToConvert", "type": "str", "optional": true, "flag": "--frameToConvert", "short_flag": "-f", "action": "store", "help": "frame to convert (for 4D data)", "default": "-1", "ui_exposed": true}, {"name": "printElapsedTime", "type": "bool", "optional": true, "flag": "--printElapsedTime", "short_flag": "--printElapsedTime", "action": "store_true", "help": "print program run time", "default": false, "ui_exposed": true}, {"name": "reslice", "type": "bool", "optional": true, "flag": "--reslice", "short_flag": "-r", "action": "store_true", "help": "save images along i, j, k direction -- 3D input only", "default": false, "ui_exposed": true}, {"name": "showSlices", "type": "bool", "optional": true, "flag": "--showSlices", "short_flag": "--showSlices", "action": "store_true", "help": "show slices that are converted", "default": false, "ui_exposed": true}, {"name": "func", "type": "str", "optional": true, "flag": "--func", "short_flag": "--func", "action": "store", "help": "apply the specified transformation function before saving", "default": "", "ui_exposed": true}, {"name": "synopsis", "type": "bool", "optional": true, "flag": "--synopsis", "short_flag": "-y", "action": "store_true", "help": "short synopsis", "default": false, "ui_exposed": true}, {"name": "convertOnlySingleDICOM", "type": "bool", "optional": true, "flag": "--convertOnlySingleDICOM", "short_flag": "--convertOnlySingleDICOM", "action": "store_true", "help": "if specified, only convert the specific input DICOM", "default": false, "ui_exposed": true}], "icon": "", "authors": "Arushi Vyas <dev@babyMRI.org>", "title": "A ChRIS plugin app", "category": "", "description": "An app to convert medical MRI image data like DICOMs and NIFTI to readable formats like PNG/JPG", "documentation": "https://github.com/FNNDSC/pl-med2img", "license": "MIT", "version": "1.1.0.1", "selfpath": "/usr/local/bin", "selfexec": "med2img", "execshell": "/usr/bin/python3", "max_number_of_workers": 1, "min_number_of_workers": 1, "max_memory_limit": "", "min_memory_limit": "", "max_cpu_limit": "", "min_cpu_limit": "", "max_gpu_limit": 0, "min_gpu_limit": 0}
```

Comment:
Can you please prettify the JSON?

Reply:
Done: https://github.com/darwinai/covidnet_integration/pull/2/commits/88daefae994d8684cb612525089c2b76b6ecc1ec